Inventory

NotJezus opened the pull request:
the basic set up for the inventory overlay

NotJezus committed:
inventory setup

made a inventory overlay, have logic in place for items in the inventory but no items

NotJezus committed:
Merge branch 'main' into inventory

NotJezus committed:
fixed inventory

inventory has been readded to the player

NotJezus committed:
sprite

forgot to add test sprite

NotJezus committed:
Merge branch 'main' into inventory

NotJezus committed:
bug fixes and refinement

deleted my comments, fixed description image not showing up and deleted an file I worked on and forgot. the file was basically a prototype of inventroyDescription

Ian-kowalski commented on Assets/Scripts/Player/MovementScript.cs:
```diff
@@ -144,7 +144,7 @@ public void jump()
 
         // Jump buffer logic
-        if (Input.GetButtonDown("Jump"))
+        if (Input.GetKeyDown(KeyCode.Space))
```
wat is the reason behind this change and do i need to do the same thing with ESC

NotJezus replied:
was just me testing for control settings. don't have to man

Ian-kowalski reviewed (changes requested):
<img width="3840" height="2160" alt="image" src="https://github.com/user-attachments/assets/e5de24ef-29ef-41c1-a8bf-647cb5cd9d81" />

scaling is off

NotJezus committed:
Inventory quick fix

canvas is now scaled to screen size and I made it so it doesn't change how it looks.
